src/components/Main/Main.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import CoffeeItem from "../CoffeeItem/CoffeeItem";
+import { bannerImages, heroImage, hotCupImage } from "../../data/main";
+import "./Main.css";
+
+class Main extends Component {
+  state = {
+    bannerImages,
```

Comment:
No need for state map the constant direct 
```suggestion
    bannerImages,
```